Userland/Libraries/LibJS/Heap/HeapBlock.cpp

```diff
@@ -20,13 +20,7 @@ namespace JS {
 
 NonnullOwnPtr<HeapBlock> HeapBlock::create_with_cell_size(Heap& heap, CellAllocator& cell_allocator, size_t cell_size)
 {
-#ifdef AK_OS_SERENITY
-    char name[64];
-    snprintf(name, sizeof(name), "LibJS: HeapBlock(%zu)", cell_size);
-#else
-    char const* name = nullptr;
-#endif
-    auto* block = static_cast<HeapBlock*>(heap.block_allocator().allocate_block(name));
+    auto* block = static_cast<HeapBlock*>(cell_allocator.block_allocator().allocate_block());
```

Comment:
This loses the neat name inside serenity that we could see either in sysfs directly or in system monitor. Was that intentionally removed, or just a casualty of the refactor?

Reply:
Fair question, that was mostly a casualty of the refactor. I can put it back :)